docs/en_US/getting-started/getting-started.md

```diff
@@ -1,113 +1,222 @@
-# Overview
+# Product overview
 
-This project provides an operator for managing EMQX clusters on Kubernetes.
+This project provides an Operator for managing EMQX clusters on Kubernetes.
 
-**Note**: EMQX Operator Controller requires Kubernetes v1.20.11 and up.
+## Deploy EMQX Operator
 
-## Running the Operator
+### Prepare the environment
 
-### Prepare
+Before deploying EMQX Operator, please confirm that the following components have been installed:
 
-We use a [cert-manager](https://github.com/jetstack/cert-manager) for provisioning the certificates for the webhook server. You can follow [the cert-manager documentation](https://cert-manager.io/docs/installation/) to install it.
+| Software                | Version Requirements |
+|:-----------------------:|:--------------------:|
+|  Kubernetes             |  >= 1.24             |          
+|  Helm                   |  >= 3                |
+|  cert-manager           |  >= 1.1.6            |
+
+- Why Kubernetes 1.24 is needed
```

Comment:
I think this should be " note ", not a "list "

Reply:
> ould be " note ", not a "list "

Sorry, I didn't understand what you mean of note, a sub title ? or can you give a example .